backend/app/app/api/endpoints/jobs.py

```diff
@@ -71,5 +71,13 @@ def update_job(
     job = jobs_service.get(db_session=db, obj_id=id)
     if not job:
         raise HTTPException(status_code=404, detail="Job not found")
+    # If current job status is completed state
+    if int(job.status_id) in [4,5,6]:
```

Comment:
🤔  - In the context of the behavior we had last week where Concourse replayed some jobs, would this end up masking that in the UI since our jobs just fire-and-forget the status updates? (e.g. the change from failed -> succeeded would eventually show up, but not the intervening state changes while the job runs)

Reply:
That's unfortunately correct. If this behavior is too broad, we can restrict it to just the aborted status.

Reply:
I don't really have a strong feeling either way -- this was mostly just a question for my own understanding.